src/admin-views/tribe-options-tickets.php

```diff
@@ -121,7 +121,15 @@
 	'ticket-attendee-modal'         => [
 		'type'            => 'checkbox_bool',
 		'label'           => esc_html__( 'Attendee Registration Modal ', 'event-tickets-plus' ),
-		'tooltip'         => esc_html__( 'Check this box if you wish to turn on the Attendee Registration Modal on the single-event page.', 'event-tickets-plus' ),
+		'tooltip' => sprintf(
+			esc_html_x(
+				'Enabling the Attendee Registration Modal provides a new sales flow for purchasing tickets that include Attendee Registration. [%sLearn more%s]',
```

Comment:
Always be using placeholder numbering

```suggestion
				'Enabling the Attendee Registration Modal provides a new sales flow for purchasing tickets that include Attendee Registration. [%1$sLearn more%2$s]',
```

Reply:
I thought of that but it's opening and closing brackets so they shouldn't be allowed to be reordered.